themes/custom/mappy/mappy.theme

```diff
@@ -12,6 +13,14 @@ function mappy_preprocess_node(&$variables) {
 }
 
 /**
+ * Implements template_preprocess_region().
+ */
+function mappy_preprocess_region(&$variables) {
+  $site_config = \Drupal::config('system.site');
+  $variables['site_name'] = $site_config->get('name');
```

Comment:
@timstallmann if we wanted to, we could skip the database layer and hardcode the site name as well. But I'm sure this is fine too.